install_scripts/ubuntu_install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+repository="palmtop/wsdd"
+
+#download wsdd python script
+rm -f wsdd.py
+wget -q https://github.com/$repository/raw/master/src/wsdd.py
+if ! [ -f wsdd.py ] ; then
+echo "Failed to download wsdd.py"
+exit
+fi
+#download wsdd systemd service file
+rm -f wsdd.service
+wget -q https://github.com/$repository/raw/master/etc/systemd/wsdd.service
+if ! [ -f wsdd.service ] ; then
+echo "Failed to download wsdd.service"
+exit
+fi
+#install wsdd python script to /usr/bin
+sudo cp wsdd.py /usr/bin/wsdd
+sudo chown root:root /usr/bin/wsdd
+sudo chmod 0755 /usr/bin/wsdd
+#install wsdd.service to systemd
+sudo cp wsdd.service /etc/systemd/system/wsdd.service
+# enable and run the service
+sudo systemctl daemon-reload
+sudo service wsdd start
+sudo systemctl enable wsdd.service
```

Comment:
The whole script is obviously distro-specific as its name suggest. Again, this is something, I really don't want to include in the source code repository, because of its distribution affinity. This has nothing to do with a particular distro, but I don't want this repo to be cluttered with specific changes. 

The second and even more important point is that a script is not a good way to install a package, which - as pointed above - is not related to a specific release of wsdd but one which is random (in time). While such a script appears easy to use, a package that refers to a specific version would be the best. And I think, the community would benefit much more from this. 

Since Ubuntu is "only" a Debian-derivative, we should aim for this. There are already according efforts going, like in #19. Such a package and ultimately the integration in the Debian/Ubuntu package repositories, would be really a good contribution.